src/core/flow.js

```diff
@@ -217,8 +217,9 @@ export function Flow({
  *   // A phase-specific movesPerTurn.
  *   movesPerTurn: integer,
  *
- *   // List of moves that are allowed in this phase.
- *   allowedMoves: ['moveA', ...],
+ *   // List of moves or a function that returns a list of moves
+ *   // that are allowed in this phase.
+ *   allowedMoves: (G, ctx) => ['moveA', ...],
```

Comment:
Let's also make this available on a per-game basis (not just on a per-phase basis)?

Reply:
Oops, forgot to wait for this. No worries, I'll just add this myself.